features/content_crop.feature

```diff
@@ -39,12 +40,7 @@ Feature: Cropping the Image Articles
       }
       """
       When we publish "123" with "publish" type and "published" state
-      Then we get response code 400
-      """
-      {
-          "_issues": {"validator exception": "[['RENDITIONS is a required field']]"}
-      }
-      """
+      Then we get response code 200
```

Comment:
Yes, vocabulary was missing from the scenario so the renditions that needed to be validated were not generated - which raised the 400 error code.